frontend/src/locales/de.json

```diff
@@ -197,13 +197,14 @@
       "uppercase": "Großbuchstabe erforderlich."
     },
     "thx": {
-      "activateEmail": "Dein Konto wurde noch nicht aktiviert. Bitte überprüfe deine E-Mail und klicke den Aktivierungslink!",
+      "activateEmail": "Dein Konto wurde noch nicht aktiviert. Bitte überprüfe deine E-Mail und klicke den Aktivierungslink! Oder fordere einen neuen Aktivierungslink über die password reset seite.",
```

Comment:
Please provide a button to the page

Reply:
Button was their their! This is the locale page so their is no button defined here

Reply:
But we don't store the email in this cases so we could set the email directly in the textfield!